pkg/packagekit/updates.jsx

```diff
@@ -17,10 +17,11 @@
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
 
-var cockpit = require("cockpit");
-var React = require("react");
-var moment = require("moment");
-var Tooltip = require("cockpit-components-tooltip.jsx").Tooltip;
+const cockpit = require("cockpit");
+const React = require("react");
+const moment = require("moment");
+const Tooltip = require("cockpit-components-tooltip.jsx").Tooltip;
+const Markdown = require("react-remarkable");
```

Comment:
If you change this, what about using `import ...`?

Reply:
Ah, can do. Seems this now conflicts after landing #7830, so I need to force-push anyway.